patches/net/minecraft/client/particle/ParticleRenderType.java.patch

```diff
@@ -0,0 +1,48 @@
+--- a/net/minecraft/client/particle/ParticleRenderType.java
++++ b/net/minecraft/client/particle/ParticleRenderType.java
+@@ -52,6 +_,11 @@
+         public String toString() {
+             return "PARTICLE_SHEET_OPAQUE";
+         }
++
++        @Override
++        public boolean isTranslucent() {
++            return false;
++        }
+     };
+     ParticleRenderType PARTICLE_SHEET_TRANSLUCENT = new ParticleRenderType() {
+         @Override
+@@ -91,6 +_,11 @@
+         public String toString() {
+             return "PARTICLE_SHEET_LIT";
+         }
++
++        @Override
++        public boolean isTranslucent() {
++            return false;
++        }
+     };
+     ParticleRenderType CUSTOM = new ParticleRenderType() {
+         @Override
+@@ -107,6 +_,11 @@
+         public String toString() {
+             return "CUSTOM";
+         }
++
++        @Override
++        public boolean isTranslucent() {
++            return false;
```

Comment:
In my testing this breaks translucency order for the item pickup particle and the elder guardian appearance particle (the only two vanilla particles using `CUSTOM`) when looking at translucent blocks such as water.